packages/vanilla/src/components/modal/modal.stories.js

```diff
@@ -1,182 +1,75 @@
-  return `
-    <button class="cbp-btn cbp-btn__primary cbp-hidden" data-modal-target="modal-three" onclick="openModal('modal-three', this)">3 Choice Modal</button>
-    <div id="modal-three" role="dialog" aria-labelledby="modal-three-label" aria-modal="true">
-      <div class="cbp-modal__header">
-        <i class="fas fa-file-alt"></i>
-        <h5 class="cbp-modal__title" id="modal-three-label">Document Changes</h5>
-      </div>
-      <hr>
-      <div class="cbp-modal__content">
-        <p id="modal-three-desc">What would you like to do with the changes you have made to this document?</p>
-      </div>
-      <div class="cbp-modal__footer cbp-modal__footer--triple">
-        <button class="cbp-btn cbp-btn__secondary" onclick="closeModal(this)">
-          <i class="fas fa-times"></i>
-          cancel
-        </button>
-        <button class="cbp-btn cbp-btn__primary" id="alt-btn" style="background-color: #2672de;">
-          <i class="fas fa-save"></i>
-          save
-        </button>
-        <button class="cbp-btn cbp-btn__primary">
-          <i class="fas fa-paper-plane"></i>
-          publish
-        </button>
-      </div>
-    </div>
-  `
-}
+const renderControls = (layout) => {
+  if (layout === 'double') {
+    return `
+      <div class="cbp-modal__controls">
+        <button type="button" class="cbp-btn cbp-btn__secondary" data-modal-close><i class="fas fa-times"></i>cancel</button>
+        <button type="button" class="cbp-btn cbp-btn__primary"><i class="fas fa-paper-plane"></i>submit</button>
+      </div>
+    `;
+  } else if (layout === 'triple') {
+    return `
+      <div class="cbp-modal__controls">
+        <button type="button" class="cbp-btn cbp-btn__secondary" data-modal-close><i class="fas fa-times"></i>cancel</button>
+        <button type="button" class="cbp-btn cbp-btn__danger" data-modal-close><i class="fas fa-trash-alt"></i>delete</button>
+        <button type="button" class="cbp-btn cbp-btn__primary"><i class="fas fa-paper-plane"></i>publish</button>
+      </div>
+    `;
+  } else {
+    return `
+      <div class="cbp-modal__controls">
+        <button type="button" class="cbp-btn cbp-btn__primary" data-modal-close><i class="fas fa-check"></i>accept</button>
+      </div>
+    `;
+  }
+};
 
-const FormModalTemplate = () => {
+const ModalTemplate = ({ heading, modalID, headingID, describedByID, controlsLayout }) => {
   return `
-    <button class="cbp-btn cbp-btn__primary cbp-hidden" data-modal-target="form-modal" onclick="openModal('form-modal', this)">Form Modal</button><br>
-    <div id="form-modal" role="dialog" role="dialog" aria-labelledby="form-modal-label" aria-modal="true">
-      <div class="cbp-modal__header">
-        <i class="fas fa-comment"></i>
-        <h5 class="cbp-modal__title" id="form-modal-label">Feedback</h5>
-      </div>
-      <hr>
+    <button type="button" class="cbp-btn cbp-btn__primary" data-modal="${modalID}">Open Model</button>
```

Comment:
"Modal", not "Model" for the button label

Reply:
fixed